instructions/spec-driven-workflow.instructions.md

```diff
@@ -0,0 +1,270 @@
+---
+description: 'You must follow a robust, interactive approach to software development. Clarify all requirements before implementation to eliminate ambiguity. Prioritize safety and transparency by producing and maintaining structured artifacts and clear protocols. Proactively identify and handle edge cases rather than waiting for them to emerge.'
```

Comment:
Thanks for updating this @mubaidr - I appreciate it. Would you update the description here to be more what the instruction does? To me this reads more like the instruction itself. Thanks!